package.json

```diff
@@ -76,7 +76,8 @@
     "Miika Hänninen <miika.hanninen@gmail.com>",
     "Kevin Goslar <kevin.goslar@gmail.com>",
     "John Krull <astrom.flux@gmail.com>",
-    "Maxim Koretskiy <mr.green.tv@gmail.com>"
+    "Maxim Koretskiy <mr.green.tv@gmail.com>",
+    "David Nimmo <dnimmo@gmail.com>"
```

Comment:
Please revert the changes to this file. We will bump the version only on new releases and will add all new contributors then.